.github/workflows/yardoc.yml

```diff
@@ -0,0 +1,35 @@
+name: Yardoc
+# check documentation in specified paths
+on: # `on` parses to `true` in ruby
+  pull_request:
+    paths:
+      # https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#patterns-to-match-file-paths
+      - modules/burials/**/*.rb
+      - modules/pensions/**/*.rb
+permissions:
+  contents: read
+  pull-requests: write
+  statuses: write
+jobs:
+  yardoc:
+    name: Yardoc
```

Comment:
You could update this one as well. I _think_ it's what shows up [here](https://github.com/department-of-veterans-affairs/vets-api/actions/runs/13139254666/job/36662099045?pr=20578):
![image](https://github.com/user-attachments/assets/4422d46a-d09c-447f-b3ad-f83aecd2272d)
It would just provide a bit more information if someone is wondering when it runs.

Reply:
I was thinking some more information could be provided here so someone didn't need to go into the workflow yaml. But based on your other comment, you want this to be generic.